master/buildbot/changes/gerritchangesource.py

```diff
@@ -387,6 +433,14 @@ def eventReceived(self, event):
             yield self.master.db.state.setState(self._oid, 'last_event_ts', event['eventCreatedOn'])
         return res
 
+    @defer.inlineCallbacks
+    def getFiles(self, change, patchset):
+        res = yield self._http.get("/changes/%s/revisions/%s/files/" % (change, patchset))
+        res = yield res.content()
+
+        res = bytes2unicode(res.splitlines()[1])  # the first line of every response is `)]}'`
```

Comment:
you can just call .decode() now that we don't have to deal with python2.6

Reply:
Good news. I would have removed it in the other places where it's called, but several of the tests  depend on the "passthrough" behaviour of `bytes2unicode`. Can fix in a follow up PR.